view/frontend/web/js/view/payment/omise.js

```diff
@@ -9,47 +9,21 @@ define(
     ) {
         'use strict';
 
-        rendererList.push(
-            {
-                type: 'omise_cc',
-                component: 'Omise_Payment/js/view/payment/method-renderer/omise-cc-method'
-            }
-        );
+        const METHOD_RENDERERS = [
+            'cc',
+            'offsite-internetbanking',
+            'offsite-alipay',
+            'offsite-installment',
+            'offsite-truemoney',
+            'offline-tesco'
+        ];
```

Comment:
maybe it could go with `omise-` prefixes and it should be just fine for 👇 ?
```js
rendererList.push({
    type: rendererName.replace(/-/g, '_'),
    component: 'Omise_Payment/js/view/payment/method-renderer/' + rendererName + '-method'
});  
```

or you prefer to reduce the repeating `omise` prefix here as well?

Reply:
I didn't want to start doing major renaming when I knew Jacek has another payment method incoming. I agree, it makes sense though